dotcom-rendering/src/web/components/ArticleLastUpdated.tsx

```diff
@@ -5,15 +5,15 @@ import { decidePalette } from '../lib/decidePalette';
 import { PulsingDot } from './PulsingDot.importable';
 import { Island } from './Island';
 
-const lastUpdatedStyles = (palette: Palette) => css`
-	${textSans.xxsmall()}
+const lastUpdatedStyles = (palette: Palette, design: ArticleDesign) => css`
+	${design === ArticleDesign.LiveBlog ? textSans.small() : textSans.xxsmall()}
```

Comment:
Does this need to cover dead blogs too?

Reply:
I _think_ that deadblogs here are the default so this reads right to me but I could be wrong and will leave it to @marjisound to confirm!

Reply:
Thanks for pointing this out, Ben suggested that we increase the LastUpdated text size for deadblogs too 👍